front-end/lib/widgets/ui/nav/controls/shared/abstract_nav_controls_widget.dart

```diff
@@ -5,10 +5,10 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 abstract class AbstractNavControlsWidget extends ConsumerWidget {
   const AbstractNavControlsWidget({super.key});
 
-  Widget getSharedNavWidget() {
+  Widget getSharedNavWidget(double serverIconSize) {
```

Comment:
Same, rend la signature de cette method avec named parameter
`getSharedNavWidget({double serverIconSize}) and adapt usages